laravel/app/EvaluationParser.php

```diff
@@ -370,21 +370,21 @@ public function findParticipantWithFindPeopleOSU($firstName, $lastName, $userTyp
         try {
             $peopleFound = json_decode(self::findPeopleOSU($firstName, $lastName)->getBody(), true);
         } catch (\Exception $e) {
-            Log::debug('Exception: Error in Find People OSU request for name (' . $name . '). Exception code: ' . $e->getCode() . ' Exception Message: ' . $e->getMessage());
+            Log::debug("Exception: Error in Find People OSU request for name ($name) Exception code:" . $e->getCode() . " Exception Message: " . $e->getMessage());
```

Comment:
can we do `"Exception: Error in Find People OSU request for name ($name) Exception code: $e->getCode() Exception Message: $e->getMessage()"` or does that not work?

Reply:
No that doesn't work which is why I left it with single quotes.